dragonfly/world/world.go

```diff
@@ -291,6 +291,18 @@ func (w *World) BreakBlock(pos BlockPos) {
 	}
 }
 
+// BreakBlockWithoutParticles breaks a block at the position passed. Unlike when setting the block at that position to air,
+// BreakBlockWithoutParticles will also update blocks around the position.
+func (w *World) BreakBlockWithoutParticles(pos BlockPos) {
```

Comment:
The name of this method is a bit meh, but I can't think of anything better right off the bat.